nixos/doc: clarify that root commands need a login shell (#534690)
NixOS/nixpkgs: nixos/doc/manual/preface.md

If you encounter problems, please report them on the [`Discourse`](https://discourse.nixos.org), the [Matrix room](https://matrix.to/#/%23nix:nixos.org), or on the [`#nixos` channel on Libera.Chat](irc://irc.libera.chat/#nixos). Alternatively, consider [contributing to this manual](#chap-contributing). Bugs should be reported in [NixOS’ GitHub issue tracker](https://github.com/NixOS/nixpkgs/issues).

::: {.note}

> Commands prefixed with `#` have to be run as root, either by logging in

Reviewer: Don't be afraid to split sentences.

Short is good.  Less words is more clarity.

````suggestion
Commands prefixed with `#` have to be run as root.

To become root in a login-shell:

```
sudo -i
``` 

```
su -
```

````

Preferably this should not be in the preface. By the time the reader reaches the upgrading chapter it's been forgotten pretty likely.

Lets just put this directly into upgrading as a prerequisite. we will de-duplicate this later.
